src/Units/Weapons/Weapon.java

```diff
@@ -22,7 +22,7 @@ public Weapon(WeaponModel model) {
 	public double getDamage(UnitModel.UnitEnum defender) {
 		if (ammo == 0 || defender == null)
 			return 0;
-		return damageChart[model.getIndex()][defender.ordinal()];
+		return damageChart[model.getIndex()][defender.ordinal()] * (model.dmgPercent/100d);
```

Comment:
Damage modifier for the unit is in UnitModel, under the name COStr; Adding another damage modifier to the weapon itself is potentially confusing and doesn't add a huge amount of flexibility.